### PR TITLE
Blue House Article, Green House article

### PR DESCRIPTION
**Pull request by 1xvc**
well github isnt allowing me to have more than 1 pr open at one time so i have to combine

### CODE REVIEW

**Comment by 1xvc** [comment]
**can somebody pls add these so i can make a new PR**

File: app/(wiki)/buildings/articles/blue_house.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Explode the [Stove](/obstacles/stove) with a gun to break the [Fridge](/obstacles/fridge) and [Small Drawer](/obstacles/small_drawer)
+- Use the skylights to check if there are players inside - but be careful, because often they may be able to see you as well. 
+- The vault, while not having good loot, is completely indestrucible *and* reflective, providing an excellent source of cover.
+ # Trivia
+ - This is the newest house added to the game in the v.19 update. 
+ - Only 5 buildings in the game have vaults - this is one of them.
+# History
+- [v.19.1](https://github.com/HasangerGames/suroi/releases/tag/v0.19.1)
+  - Changes to vault loot
+- [v.19](https://github.com/HasangerGames/suroi/releases/tag/v0.19)
```
Comment (pgund) [inline app/(wiki)/buildings/articles/blue_house.md:21]: 0.19.0

Comment (1xvc) [reply app/(wiki)/buildings/articles/blue_house.md:21]: the loot changed in .19.0
or .19.1

Comment (1xvc) [reply app/(wiki)/buildings/articles/blue_house.md:21]: @pgund

**Pull request merged**